docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,489 @@
+  
+# Dapr 0.11.0
+
+It's been seven weeks, and We're happy to announce the release of Dapr v0.11.0!
+
+With Dapr approaching v1.0, our releases increasingly focus on operational concerns like stability, reliability, and upgradability in production environments.
```

Comment:
Let's leave future release schedule out of the release notes. Just start with "The Dapr releases increasingly focus ..."